### ISSUE TITLE
A11y_AzureToolsStorageExplorer_PropertiesTab_ScreenReader: Unnecessary Table is used for the content present under "Properties" tab.

### ISSUE THREAD

**Issue by Suraj1200499** [microsoft/AzureStorageExplorer#1922]
“[Check out Accessibility Insights!](https://accessibilityinsights.io/) Identify accessibility bugs before check-in and make bug fixing faster and easier.”

GitHubTags: #A11y_AzureToolsStorageExplorer; #A11yMAS; #A11YTCS; #DesktopApp; #MAS1.3.1; #NVDA; #JAWS; #Win32; 

**Environment Details:**
Application Name: Azure Tools Storage Explorer
Application Version: 1.10.0
Windows10

**Repro Steps:**
1. Launch Storage Explorer
2. Navigate to "Open Connect dialog" pane and Sign in to Azure.
3. Go to "Manage Accounts" pane and Select the subscriptions you will use. Once selected, click on "Apply" button.
4. Navigate to "Blob Containers" of the selected subscription, right click and hit enter on "Create Blob container" to Create a new Blob container.
5. Navigate to "Properties" Tab present on left bottom of the screen and hit enter.
6. Start NVDA.
7. Navigate using Tab key to content present under "Properties" Tab, verify NVDA unnecessary announces "rows and columns" for the contents.

**​Actual:​**
Unnecessary Table is used for the contents present under "Properties" Tab because of Table used NVDA Unnecessary announces "rows and columns" for the contents.

Same issue is observed in "**JAWS**" also.

**Same issue is observed in below Scenarios:**
1. **Blob  Container Properties tab**

**Expected:**
Table should not be used for the contents present under "Properties" Tab and also NVDA should not announce "rows and columns" for the contents present under "Properties" Tab

**UserImpact:**
Screen reader users gets confusion if screen reader unnecessary announces "rows and columns" for the content present under "properties" tab.

**Recommendation:**
https://microsoft.sharepoint.com/teams/msenable/mas/Pages/browse-fixes.aspx

**MAS Reference:**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4}

**Attachment for Reference:**
[A11y_AzureToolsStorageExplorer_PropertiesTab_ScreenReader.pptx](https://github.com/microsoft/AzureStorageExplorer/files/3656592/A11y_AzureToolsStorageExplorer_PropertiesTab_ScreenReader.pptx)

**Comment by MRayermannMSFT** [comment]
Fix merged into master. We're now not using <table> to construct the properties grid.

**Comment by MRayermannMSFT** [comment]
@Radha2019 @Charanrajkb @nirmalsahoo @Suraj1200499 @vinodbk197

Please verify whether or not this bug was fixed to your satisfaction in the latest release of Storage Explorer. You can obtain that release from https://www.storageexplorer.com

If the bug has not been fixed to your satisfaction, please:
- Explain how did the fix not meet your expectations.
- Tell us exactly how we can meet your expectations. If there is more than one way to do so, feel free to list the different ways.

**Comment by vinodbk197** [comment]
Verified this issue on:Azure Tools Storage Explorer (Version: 1.12.0)
The issue is no longer to reproduces, Hence closing this bug.

**Comment by fsteffi** [comment]
GithubTags:#Closed;

**Comment by mstechie** [comment]
GitHubTags:#BM-TCS-StorageExplorer-Win32-Sep2019;
